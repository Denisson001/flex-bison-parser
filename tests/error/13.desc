Compilation error - invalid string expression