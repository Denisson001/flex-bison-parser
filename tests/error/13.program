{
    a = 5;
    b_s = "123";
    a_s = b_s + b_s + b_s * a;
}