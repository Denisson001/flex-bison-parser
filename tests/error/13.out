Parser: —Åompilation error - syntax error, unexpected '*', expecting ';' or '+' near line 4
